core/src/main/java/org/jdbi/v3/core/statement/StatementContext.java

```diff
@@ -475,26 +478,71 @@ public void setConcurrentUpdatable(final boolean concurrentUpdatable) {
         this.concurrentUpdatable = concurrentUpdatable;
     }
 
+    /**
+     * @return the {@link Instant} at which query execution began
+     */
+    @Nullable
     public Instant getExecutionMoment() {
         return executionMoment;
     }
 
+    /**
+     * @return the {@link ZonedDateTime} at which query execution began
+     */
+    @Nullable
+    public ZonedDateTime getExecutionTimestamp() {
+        return executionMoment == null ? null : executionMoment.atZone(getConfig(Time.class).getClock().getZone());
+    }
```

Comment:
I think we could ditch these overloads. If the user wants them rendered converted to a zone they can do that themselves.

Reply:
Well that was kind of the entire point of adding the Clock inside jdbi, that the whole process would be more convenient.

But I pretty much agree with steven by now, this PR is dead (long live this PR)